src/main/java/tuteez/model/person/lesson/LessonManager.java

```diff
@@ -82,9 +77,7 @@ public boolean isExistingLesson(Lesson lesson) {
      */
     public Map<Person, ArrayList<Lesson>> getClashingLessons(UniquePersonList studentList, Lesson lesson) {
         Map<Person, ArrayList<Lesson>> clashingLessonMap = new HashMap<>();
-        Iterator<Person> students = studentList.iterator();
-        while (students.hasNext()) {
-            Person studentToCheck = students.next();
+        for (Person studentToCheck : studentList) {
             ArrayList<Lesson> clashedLessons = studentToCheck.getLessonsThatClash(lesson);
```

Comment:
I feel like to an outsider, it is not entirely clear how getClashingLessons and getLessonsThatClash differ as two separate methods, consider renaming as getAllClashingLessons and/or getClashingLessonsFromStudent?

Reply:
Oops idky this ended up on this line but ugwim right

Reply:
Agree. I will change the names soon!!